src/file.helpers.ts

```diff
@@ -1,16 +1,19 @@
+import { readFileSync } from 'fs';
+
 import { getProjectDirectoryPath } from './path.helpers';
 import { LxrConfig, PackageJson } from './interfaces';
 
+export function readJsonFile(path: string): unknown {
+  const buffer = readFileSync(path);
+  return JSON.parse(buffer.toString('utf-8'));
+}
+
 export function loadLxrConfig(): LxrConfig {
   const lxrConfigPath = getProjectDirectoryPath('lxr.json')
-
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  return require(lxrConfigPath);
```

Comment:
I was actually looking for alternatives for loading a JSON file without using require, but I didn't find anything. Cool! 👍